internal/resources/fabric/service_token/resource_test.go

```diff
@@ -63,7 +63,54 @@ func TestAccFabricServiceToken_PNFV(t *testing.T) {
 	})
 }
 
-func testAccFabricServiceTokenConfig(serviceTokenName string, serviceTokenDescription string, virtualDeviceUuid string) string {
+func TestAccFabricAsidePortServiceToken_PNFV(t *testing.T) {
+	ports := testing_helpers.GetFabricEnvPorts(t)
+	var portUuid string
+	if len(ports) > 0 {
+		portUuid = ports["ppds"]["dot1q"][0].GetUuid()
+	}
+	serviceTokenName, serviceTokenUpdatedName := "token_port_PNFV", "UP_Token_port_PNFV"
```

Comment:
Update suffixes to match actual user account used. Looks like a different user is being used here based onthe portUuid being pulled from the PPDS sub portion.